ironic/templates/configmaps/nginx-default.conf.yaml.j2

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  default.conf: |+
```

Comment:
it's done by default in nginx:alpine :
```
/ # ls -lah /var/log/nginx/error.log 
lrwxrwxrwx    1 root     root          11 Sep 23 17:11 /var/log/nginx/error.log -> /dev/stderr
/ # ls -lah /var/log/nginx/access.log 
lrwxrwxrwx    1 root     root          11 Sep 23 17:11 /var/log/nginx/access.log -> /dev/stdout
```

let me know if this is not the solution you expect